docs/reference/index.rst

```diff
@@ -35,3 +35,7 @@ The MIOpen API library is structured as follows:
   * :doc:`ReduceExtreme <../doxygen/html/group__ReduceExtreme>` (experimental)
   * :doc:`Getitem <../doxygen/html/group__getitem>` (experimental)
   * :doc:`ReduceCalculation <../doxygen/html/group__ReduceCalculation>` (experimental)
+  * :doc:`RotaryPositionalEmbeddings <../doxygen/html/group__RotaryPositionalEmbeddings>` (experimental)
+  * :doc:`ReLU <../doxygen/html/group___re_l_u>` (experimental)
+  * :doc:`GLU <../doxygen/html/group__glu>` (experimental)
```

Comment:
sync with develop branch so that it is easier to track your changes

Reply:
help me to sync

Reply:
did sync it